tests/components/auth/test_init.py

```diff
@@ -607,17 +631,26 @@ async def test_ws_delete_all_refresh_tokens(
     tokens = result["result"]
 
     await ws_client.send_json(
-        {
-            "id": 6,
-            "type": "auth/delete_all_refresh_tokens",
-        }
+        {"id": 6, "type": "auth/delete_all_refresh_tokens", **delete_token_type}
     )
 
     result = await ws_client.receive_json()
     assert result, result["success"]
+    remaining_tokens = 0
     for token in tokens:
         refresh_token = await hass.auth.async_get_refresh_token(token["id"])
-        assert refresh_token is None
+        if refresh_token is not None:
```

Comment:
It's better 😁... but there is still some if branching that should be params

Reply:
I'm trying 🤣  

It's not obvious to me what should be the param here. Do I have to param an array of every single refresh token that I expect to get back from the async_get? That seems cumbersome.

Reply:
When struggling with something like that, its usually a sign that its better to split it into two tests